11-Backbonejs/03-Backbone-Models.md

```diff
@@ -49,7 +50,7 @@ So if the script above is run the console will result in:
 A new person has been instantiated.
 ```
 
-The defaults property lets you set default values to attributes for your model.  You can then retrieve attribute values with the `get` function.  
+The defaults property lets you set default values to **attributes** for your model.  You can then retrieve those  attribute values with the `get` function.  We add additional data to the object we pass in to `extend` by comma-separating those values.
```

Comment:
We should say "We can also provide other properties to the model by including them in the object we pass to `extend`."